### ISSUE TITLE
Failed to pull image "ywk253100/clair-photon:v2.0.1-v1.5.0-chart-patch"

### ISSUE THREAD

**Issue by mJace** [vmware/harbor#5295]
In master e415d56,
helm install failed.
![image](https://user-images.githubusercontent.com/9836691/42615981-162f5e9e-85df-11e8-8208-d515f797fbeb.png)

` kubectl describe po my-release-harbor-clair-5d45f8b964-mqnjg`
![image](https://user-images.githubusercontent.com/9836691/42615997-27e095ae-85df-11e8-9128-c1d4d3972fdf.png)

**Comment by ywk253100** [comment]
The chart on the master branch is under development. Use this one https://github.com/vmware/harbor/tree/release-1.5.0-chart-perm-fix/contrib/helm/harbor instead. But note that maybe some features don't work well. A better support for Harbor chart will be available in release 1.6.

**Comment by mJace** [comment]
Is this issue https://github.com/vmware/harbor/issues/4496 still in branch release-1.5.0-chart-perm-fix ?
I get this log after install the chart
```bash
kubectl get po
NAME                                            READY     STATUS             RESTARTS   AGE
my-release-harbor-adminserver-0                 1/1       Running            0          14m
my-release-harbor-clair-6cdfbdf9b7-gnzpr        0/1       CrashLoopBackOff   7          14m
my-release-harbor-jobservice-5d5557d8db-xqtmd   1/1       Running            1          14m
my-release-harbor-mysql-0                       1/1       Running            0          14m
my-release-harbor-registry-0                    0/1       CrashLoopBackOff   7          14m
my-release-harbor-ui-75b6867b5f-hhx9k           1/1       Running            0          14m
my-release-postgresql-5c96554c8f-xkld8          1/1       Running            0          14m
my-release-redis-master-0                       1/1       Running            0          14m
my-release-redis-slave-8476cdf8d-rscxp          1/1       Running            0          14m
```  


```bash
jace@ubuntu:~/harbor/contrib/helm/harbor$ kubectl logs my-release-harbor-registry-0
chown: changing ownership of '/etc/registry/root.crt': Read-only file system
chown: changing ownership of '/etc/registry/config.yml': Read-only file system
jace@ubuntu:~/harbor/contrib/helm/harbor$ kubectl logs my-release-harbor-clair-6cdfbdf9b7-gnzpr
chown: changing ownership of '/config/config.yaml': Read-only file system
```

**Comment by reasonerjt** [comment]
correct, it will never be merged to release 1.5.0
